### ISSUE TITLE
SCTP support for Packet Sender

### ISSUE THREAD

**Issue by dannagle** [dannagle/PacketSender#372]
### Discussed in https://github.com/dannagle/PacketSender/discussions/371

It would be nice if Packet Sender supported other transport protocols besides UDP and TCP.

**Comment by amirshahhoseini** [comment]
Yes.the SCTP is most useful  transport protocol in telecomm core
